web/admin.py

```diff
@@ -211,8 +211,18 @@ class EndpointAdmin(admin.ModelAdmin):
     ordering = ('id',)
 
 
+class ReadOnlyAdmin(admin.ModelAdmin):
+    list_display = ['listing']
+
+    def has_add_permission(self, request, obj=None):
+        return False
+
+    def has_change_permission(self, request, obj=None):
+        return False
+
 @admin.register(Exposure)
-class ExposureAdmin(admin.ModelAdmin):
+class ExposureAdmin(ReadOnlyAdmin):
     list_display = ('dataset', 'id', 'endpoint', 'form_id', 'form_name')
+    list_filter = ('endpoint__name', 'form_id', 'form_name')
```

Comment:
Not sure filtering via `form_id` will be used. It should filter the same way as `form_name` and it is less readable I think.

Reply:
i will remove it then